services/shotgrid_common/utils.py

```diff
@@ -117,6 +117,7 @@ def _sg_to_ay_dict(
     ay_entity_type = "folder"
     task_type = None
     folder_type = None
+    exception_attribs = {"status", "assignees", "tags"}
```

Comment:
I would use something like `root_attributes`. The `exception_attribs` sounds "different" from what it is.